build-system/tasks/bundle-size.js

```diff
@@ -22,7 +22,7 @@ const log = require('fancy-log');
 const {getStdout} = require('../exec');
 
 const runtimeFile = './dist/v0.js';
-const maxSize = '80.85KB';
+const maxSize = '80.9KB';
```

Comment:
Why did this PR have a bundle size impact?

Reply:
hmm good question. Perhaps the capability enum removal. Introducing the enum shorthanded the usage of the 'viewerRenderTemplate' in the codebase.

Reply:
though i can't imagine it accounting for 5kb of data.